packages/web-components/src/dapp-wallet-bridge/DappWalletBridge.js

```diff
@@ -22,7 +22,7 @@ export const BridgeProtocol = /** @type {const} */ ({
  * @typedef {{
  * instanceHandle: import('@endo/marshal').CapData<"Instance">;
  * publicInvitationMaker: string;
- * proposalTemplate: Partial<ProposalRecord>
+ * proposalTemplate: unknown
```

Comment:
the early type is correct. unfortunately the type isn't available here. I think it can be included by importing the module though to get its ambient types.
https://github.com/Agoric/agoric-sdk/blob/f74b6b55e12b30d47e342071a9fb8cf8e1b0b472/packages/zoe/src/zoeService/types.js#L221-L230

Reply:
Got it, thanks. I wasn't able to find it myself in github search, and yea it was giving a type error without the ambient import